Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -123,6 +128,10 @@ public final class ManifestLoader: ManifestLoaderProtocol {
 
         self.databaseCacheDir = cacheDir.map(resolveSymlinks)
 
+        // this queue and semaphore is used to limit the amount of concurrent manifest loading taking place
+        self.evaluationQueue = OperationQueue()
+        self.evaluationQueue.name = "org.swift.swiftpm.manifest-loader"
+        self.evaluationQueue.maxConcurrentOperationCount = Concurrency.maxOperations
         self.concurrencySemaphore = DispatchSemaphore(value: Concurrency.maxOperations)
```

Comment:
Similar question as in the HTTP PR:  if this is the same concurrency value, what does the semaphore provide that the queue limit doesn't?

Reply:
Got my answer from that PR: when the operation being performed by the queue is async, the semaphore is needed in order for the operation queue limit to be effective.